package.json

```diff
@@ -36,6 +36,9 @@
     "mocha": "^4.0.1",
     "semantic-release": "^12.4.1"
   },
+  "peerDependencies" : {
+    "eslint-plugin-react": "^7.11.1"
```

Comment:
i'd suggest making it both a peer and a dep; that way the requirement hoists, but users won't necessarily need to install it if they're not using it :-)

Reply:
Yeap, it is in both now.